tests/foreman/smoke/test_api_smoke.py

```diff
@@ -995,13 +1009,7 @@ def _search(self, entity, query, auth=None):
             verify=False,
         )
         response.raise_for_status()
-        self.assertEqual(
```

Comment:
This assertion is not useful as the API just reproduce the search term. By reading the assertion message it was intended to check if the search returned a result or not, which was not ensured at all.